scss/_patterns_media-container.scss

```
@@ -0,0 +1,5 @@
+@mixin vf-p-media-container {
+  .p-media-container {
+    margin-top: $sp-unit;
```

This would be more accurate variable to use I think. The value is the same, but `$sp-unit` is more for calculations rather than direct usage in components.

```suggestion
    margin-top: $spv--small;
```